src/types/group.rs

```diff
@@ -0,0 +1,51 @@
+use chrono::{DateTime, Utc};
+use serde::{Deserialize, Serialize};
+
+#[derive(PartialEq, Eq, PartialOrd, Ord, Hash, Clone, Debug, Serialize, Deserialize)]
+pub struct CreateGroupRequest {
+    pub group_name: String,
+}
+
+#[derive(PartialEq, Eq, PartialOrd, Ord, Hash, Clone, Debug, Serialize, Deserialize)]
+pub struct CreateGroupResponse {
+    pub group_name: String,
+    pub owner_email: String,
+}
+
+#[derive(PartialEq, Eq, PartialOrd, Ord, Hash, Clone, Debug, Serialize, Deserialize)]
+pub struct AddUserToGroupRequest {
+    pub user_email: String,
+}
+
+#[derive(PartialEq, Eq, PartialOrd, Ord, Hash, Clone, Debug, Serialize, Deserialize)]
+pub struct KickUserFromGroupRequest {
+    pub user_email: String,
+}
+
+#[derive(PartialEq, Eq, PartialOrd, Ord, Hash, Clone, Debug, Serialize, Deserialize)]
+
+pub struct UserGroup {
+    pub created_at: DateTime<Utc>,
+    pub last_modified: DateTime<Utc>,
+    pub owner_email: String,
+    pub group_name: String,
+    pub is_admin: Option<bool>, //only present in the relatively uncommon case that the user is an admin
```

Comment:
Is there a reason why you decided against using default here to make this just a `bool`?

Reply:
Just my preconceived notions from other json frameworks. I'll do that 👍

Reply:
Done in https://github.com/phylum-dev/phylum-types/pull/23/commits/e533d922f9ffd5737eabeb59fbb98b1f9d8caeae